src/modules/rlm_ldap/user.c

```diff
@@ -221,20 +221,24 @@ rlm_rcode_t rlm_ldap_check_access(rlm_ldap_t const *inst, request_t *request, LD
 void rlm_ldap_check_reply(module_ctx_t const *mctx, request_t *request, fr_ldap_thread_trunk_t const *ttrunk)
 {
 	rlm_ldap_t	*inst = talloc_get_type_abort(mctx->inst->data, rlm_ldap_t);
+	fr_pair_t 	*parent;
 
-       /*
+   /*
 	*	More warning messages for people who can't be bothered to read the documentation.
 	*
 	*	Expect_password is set when we process the mapping, and is only true if there was a mapping between
 	*	an LDAP attribute and a password reference attribute in the control list.
 	*/
 	if (!inst->expect_password || !RDEBUG_ENABLED2) return;
 
-	if (!fr_pair_find_by_da(&request->control_pairs, NULL, attr_cleartext_password) &&
-	    !fr_pair_find_by_da(&request->control_pairs, NULL, attr_nt_password) &&
+	parent = fr_pair_find_by_da_nested(&request->control_pairs, NULL, attr_password);
+	if (!parent) parent = request->control_ctx;
```

Comment:
As I said before, this likely won't work until we implement the conversion fully